cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java

```diff
@@ -384,11 +406,18 @@ public String getValuesForInsert() {
             
                 while (it.hasNext()) {
                     ArrayList<String> values = (ArrayList<String>) aggregation.get((String) it.next());
+                    String valueType = (String) it.next();
+                    String value = values.get(i);
+                    if (value == null) { // TBD check also if value == "" ?
+                        value = "NULL";
+                    } else if (!(attrNativeTypes && valueType.equals("Number"))) {
```

Comment:
Its not just a matter of NGSI Type, but also related to REAL Type (NGSIv2 JSON native types).